scala-effect-api/src/main/scala/com/avast/metrics/scalaeffectapi/TimerPair.scala

```diff
@@ -0,0 +1,15 @@
+package com.avast.metrics.scalaeffectapi
+
+import java.time.Duration
+
+trait TimerPair[F[_]] {
+  trait TimerPairContext {
+    def stop: F[Duration]
+    def stopFailure: F[Duration]
+  }
+
+  def start: F[TimerPairContext]
+  def stop(context: TimerPairContext): F[Duration]
+  def stopFailure(context: TimerPairContext): F[Duration]
```

Comment:
This is inconsistent with the other monitors, e.g.
```java
    TimeContext start();
    void update(Duration duration);
    void updateFailure(Duration duration);
```
The `stop` methods are already in the `TimerPairContext`.